lib/app.js

```diff
@@ -102,7 +104,7 @@ const TermsOfStorage = () => (
 )
 
 // eslint-disable-next-line complexity
-const SvgIcon = ({ className, iconName, style }) => {
+const SvgIcon = ({ className, FallbackIcon, iconName, style }) => {
```

Comment:
Can we call this `Fallback` instead of `FallbackIcon`? Since you can send any jsx you like
```suggestion
const SvgIcon = ({ className, Fallback, iconName, style }) => {
```